test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -83,12 +83,14 @@ namespace test
 {
 
 boost::random::mt19937 RandomCode::gen;
+boostIntDistrib RandomCode::prcentDist = boostIntDistrib (0, 100);
 boostIntDistrib RandomCode::opCodeDist = boostIntDistrib (0, 255);
 boostIntDistrib RandomCode::opLengDist = boostIntDistrib (1, 32);
 boostIntDistrib RandomCode::opMemrDist = boostIntDistrib (0, 10485760);
 boostIntDistrib RandomCode::uniIntDist = boostIntDistrib (0, 0x7fffffff);
 boostUint64 RandomCode::uInt64Dist = boostUint64 (0, std::numeric_limits<uint64_t>::max());
 
+boostIntGenerator RandomCode::randPrcentGen = boostIntGenerator(gen, prcentDist);
```

Comment:
`randPercentGen`.  Somehow I sense this was done to keep the widths uniform, but it's anyway not always possible to keep the same number of characters.